next.config.js

```diff
@@ -62,6 +62,25 @@ const moduleExports = withBundleAnalyzer({
 		defaultLocale: 'en',
 		localeDetection: false,
 	},
+	headers: () => {
+		return [
+			{
+				// Adding CORS headers for /manifest.json
+				source: '/manifest.json',
+				headers: [
+					{
+						key: 'Access-Control-Allow-Origin',
+						value: 'https://app.safe.global',
+					},
+					{ key: 'Access-Control-Allow-Methods', value: 'GET' },
```

Comment:
dont we need other methods support too ? GET,DELETE,PATCH,POST,PUT

Reply:
Nope, it's only for the dev environment to work inside safe's interface - we could add those methods but it's not needed